src/app/auth/sign-up/sign-up.component.html

```diff
@@ -0,0 +1,124 @@
+<div class="row progress-row">
+  <div class="col-4 sections-col" *ngFor="let sectionIcon of sectionsIcons; let i = index">
+    <div class="row">
+      <div class="col-6">
```

Comment:
Please try to avoid bootstrap grid, instead of this please use `flex`

Reply:
Rewrote using display:flex